### PR TITLE
Use textarea for query header to make it easier to editing long title

### PR DESCRIPTION
**Pull request by argonism**
This is a working code for https://github.com/bdash-app/bdash/issues/274

With this implementation, query title will behave like this:

https://github.com/user-attachments/assets/a9942248-7b22-40a9-b4b7-2d90d837b71d

### CODE REVIEW

**Comment by argonism** [comment]
@hokaccha  @morishin
Hi, excuse me for mentioning all the maintainer-like people together.
Would you mind taking a look at this if you like?

File: src/renderer/components/QueryHeader/QueryHeader.css
Comment (argonism) [inline src/renderer/components/QueryHeader/QueryHeader.css:9]: Use flexbox over position-absolute not to overlap between query header title and select element.

**Review by morishin — APPROVED** [review]
sorry for late!

**Comment by argonism** [comment]
Thank you for review! Is there anything I can do for further process?
I can see that 1 workflow is awaiting to be approval but I can't approve it.

**Pull request merged**
